File: m/rls/stskp.r
file	stskp - string skip whitespace
include	rid:rider
include	rid:stdef

code	st_skp - string skip whitespace

  func	st_skp
	str : * Char			; string
	()  : * char			;
  is	while *str eq ' '		; whitestuff
	   || *str eq '\t'		;
	.. ++str			; skip it
	reply str			; send back result
  end


File: m/rls/rlb/stskp.c
/* file -  stskp - string skip whitespace */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
/* code -  st_skp - string skip whitespace */
char *st_skp(
register char *str )
{ while (( *str == ' ')
  ||(*str == '\t')) {
    ++str; }
  return ( str);
} 
